ext/json/json_arginfo.h

```diff
@@ -5,6 +5,7 @@ ZEND_BEGIN_ARG_WITH_RETURN_TYPE_MASK_EX(arginfo_json_encode, 0, 1, MAY_BE_STRING
 	ZEND_ARG_TYPE_INFO(0, value, IS_MIXED, 0)
 	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, flags, IS_LONG, 0, "0")
 	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, depth, IS_LONG, 0, "512")
+	ZEND_ARG_TYPE_MASK(0, indent, MAY_BE_STRING|MAY_BE_LONG, "4")
```

Comment:
If I'm not mistaken, I believe this file should be generated from `./json.stub.php` using `/build/gen_stub.php`

Reply:
Good catch, I updated the stub file and updated the arginfo file!